client/X11/xf_floatbar.c

```diff
@@ -474,6 +503,8 @@ static void xf_floatbar_button_update_positon(xfFloatbar* floatbar)
 				break;
 		}
 
+		WINPR_ASSERT(xfc);
```

Comment:
Does it really make sense to ASSERT this in every loop iteration?

Reply:
dereferencing here, and it is not a high performance loop, just over the buttons